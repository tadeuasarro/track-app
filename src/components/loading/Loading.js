import './loading.css';
import Navbar from '../navbar/Navbar';

const Loading = () => (
  <div>
    <Navbar />
    <div className="loading-container">
      <h1>Please wait...</h1>
    </div>
  </div>
);

export default Loading;
