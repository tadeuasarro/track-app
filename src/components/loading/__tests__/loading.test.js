import { render, screen, act } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Loading from '../Loading';

describe('Rendering component', () => {
  it('creates an App component', () => {
    act(() => {
      render(
        <Loading />,
      );
    });
    // eslint-disable-next-line
    const element = screen.getByText('Please wait...');
  });
});

describe('Display', () => {
  it('renders correctly', () => {
    const comp = renderer.create(
      <Loading />,
    ).toJSON();
    expect(comp).toMatchSnapshot();
  });
});
