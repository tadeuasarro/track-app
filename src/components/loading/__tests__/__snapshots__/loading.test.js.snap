// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display renders correctly 1`] = `
<div>
  <div
    className="navbar"
  >
    <h1>
      Personal Expenses
    </h1>
  </div>
  <div
    className="loading-container"
  >
    <h1>
      Please wait...
    </h1>
  </div>
</div>
`;
